crates/distribution-filename/Cargo.toml

```diff
@@ -20,7 +20,7 @@ pep440_rs = { workspace = true }
 platform-tags = { workspace = true }
 uv-normalize = { workspace = true }
 
-rkyv = { workspace = true, features = ["strict", "validation"], optional = true }
+rkyv = { workspace = true, optional = true }
```

Comment:
rkyv depends on an old version of hashbrowns, but the next release will "probably take months" (https://github.com/rkyv/rkyv/issues/458#issuecomment-1959686122) and main still depends of hashbrown for std unconditionally.